examples/ios/objc/Draw/Views/DrawView.m

```diff
@@ -0,0 +1,188 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2014 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+#import "DrawView.h"
+#import "DrawPath.h"
+#import "SwatchesView.h"
+#import "SwatchColor.h"
+#import "CanvasView.h"
+#import <Realm/Realm.h>
+
+@interface DrawView ()
+
+@property DrawPath *drawPath;
+@property NSString *pathID;
+@property NSMutableSet *drawnPathIDs;
+@property RLMResults *paths;
+@property RLMNotificationToken *notificationToken;
+@property NSString *vendorID;
```

Comment:
What's this `vendorID` property ever used for? It only seems to ever be set, never read.